src/desktop/apps/collections/client.js

```diff
@@ -0,0 +1,21 @@
+import { buildClientApp } from "reaction/Artsy/Router/client"
+import { data as sd } from "sharify"
+import { routes } from "reaction/Apps/Collections/routes"
+import mediator from "desktop/lib/mediator.coffee"
+import React from "react"
+import ReactDOM from "react-dom"
+
+buildClientApp({ routes, user: sd.CURRENT_USER })
+  .then(({ ClientApp }) => {
+    ReactDOM.hydrate(
+      <ClientApp mediator={mediator} />,
```

Comment:
This pattern changed a few months back -- `mediator` is also passed into `context`.

Reply:
(Anything you want accessible in `SystemContext` is passed into `context`)